web/css/doppelkopf.css

```diff
@@ -0,0 +1,26 @@
+.mainArea {
+  width: 1000px;
+  padding: 10px;
+  border: 1px solid black;
+}
+fieldset {
+  padding: 10px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+.schreibTabelle{
+  border-collapse:collapse;
+	border:1px solid black;
+}
+.schreibTabelle th,td{
+  width: 100px;
+  height: 25px;
+	border:1px solid black;
+	padding:5px;
+}
+.spieler{
```

Comment:
Egal wo das Projekt läuft, ist es oft best practice alle Variablen, Klassen usw. Englisch zu benennen - das wird in der Uni manchmal falsch gehandhabt. Oft ist es dann auch einfacher das mit dem Code zu vermischen, da dort ja zB alle Keywords auf Englisch sind. Oben heißt eine Klasse ja auch `mainArea`, danach dann `schreibTabelle`.